packages/neo-one-client/src/__tests__/DeveloperClient.test.ts

```diff
@@ -1,16 +1,39 @@
 import { DeveloperClient } from '../DeveloperClient';
-import { NEOONEDataProvider } from '../provider/neoone/NEOONEDataProvider';
+import { NEOONEProvider } from '../provider';
+import { LocalKeyStore, LocalMemoryStore, LocalUserAccountProvider } from '../user';
 
 describe('DeveloperClient', () => {
-  const provider = new NEOONEDataProvider({
-    network: 'net',
-    rpcURL: 'rpc',
+  const networkName = 'net';
+  const neoOneProvider = new NEOONEProvider({
+    options: [
+      {
+        network: networkName,
+        rpcURL: 'rpc',
+      },
+    ],
   });
+  const keystore = new LocalKeyStore({
+    store: new LocalMemoryStore(),
+  });
+
+  const localUserAccountProvider = new LocalUserAccountProvider({
+    keystore,
+    provider: neoOneProvider,
+  });
+
+  const providers = {
+    memory: localUserAccountProvider,
+  };
+
+  const provider = neoOneProvider.read(networkName);
 
-  const developerClient = new DeveloperClient(provider);
+  const developerClient = new DeveloperClient(provider, providers);
 
   test('runConsensusNow', async () => {
+    // tslint:disable-next-line no-object-mutation
     provider.runConsensusNow = jest.fn();
+
+    // tslint:disable-next-line no-void-expression
```

Comment:
These should just be fixed (the `no-void-expression` ones). The `no-object-mutation`s are ok to ignore.